src/server/internalServer.cpp

```diff
@@ -483,6 +493,10 @@ bool InternalServer::start() {
       std::cerr << "IP address " << addr << " is not a valid ip address." << std::endl;
       return false;
     } 
+    if (!ipAvailable(addr)) {
+      std::cerr << "IP address " << addr << " is not available on this system." << std::endl;
```

Comment:
Change the error format to "ERROR: IP address is not available on this system: *HERE_COMES_THE_IP_ADDRESS*"